ccic/bin/process.py

```diff
@@ -583,4 +590,8 @@ def run(args):
                 proc for proc in processing_processes if proc.is_alive()
             ]
 
+    processing_queue.get()
+    processing_queue.task_done()
+    processing_queue.join()
```

Comment:
I think this should work as expected, but since now I had a closer look at the code:
I understand this is necessary due to line 240-241. I cannot figure out/remember how `processing_queue.put(None)` is useful there; I think it could be omitted and just have `processing_queue.join()` here.
I also understand now that the fact that not calling `download_queue.task_done()` after `download_queue.get()` yet that it had worked well so far has to do with that we used only 1 download thread.
If you see anything wrong with these argument let me know and maybe we can improve the code (I'm careful because of #67)

Reply:
The ``put(None)``  is used as a flag to signal the processing processes to stop running. Otherwise they will run indefinitely and the program won't terminate properly.

You're right, though, that this would cause issues if there's more than on download thread. The get and task_done would need to be called as long as there will be items in the queue.